cluster-autoscaler/README.md

```diff
@@ -134,6 +134,7 @@ on them) and set a `priorityClassName: system-cluster-critical` property on your
 Supported cloud providers:
 * GCE https://kubernetes.io/docs/concepts/cluster-administration/cluster-management/
 * GKE https://cloud.google.com/container-engine/docs/cluster-autoscaler
+* IBM Cloud https://cloud.ibm.com/docs/containers?topic=containers-ca
```

Comment:
Please add information that IBM Cloud is supported outside the main repo.

Reply:
What do you mean by "supported outside the main repo"? Maybe I'm just reading the sentence wrong, but I don't follow.

Reply:
There is no IBM cloudprovider implementation in this repository. Our official image cannot run on IBM cloud. There must be some sort of a fork somewhere that contains the IBM specific code.